db/migrate/20170614165059_add_user_id_to_audio_files.rb

```diff
@@ -0,0 +1,5 @@
+class AddUserIdToAudioFiles < ActiveRecord::Migration[5.0]
+  def change
+    add_column :audio_files, :user_id, :integer
```

Comment:
Consider adding a foreign key for this as well as indexing. - https://robots.thoughtbot.com/referential-integrity-with-foreign-keys